src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs

```diff
@@ -9,15 +9,33 @@
     using CSF.SRDashboard.Client.Services.Document;
     using CSF.SRDashboard.Client.Utilities;
     using Microsoft.AspNetCore.Components;
+    using Microsoft.AspNetCore.WebUtilities;
     using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
+    using CSF.SRDashboard.Client.Components.Tables.WorkloadRequest.Entities;
+    using DSD.MSS.Blazor.Components.Core.Constants;
+    using Microsoft.JSInterop;
+
     public partial class SeafarerProfile
     {
        
         [Parameter]
         public string Cdn { get; set; }
+
+        [Parameter]
+        public int requestId { get; set; }
```

Comment:
Public properties start with an uppercase.
```suggestion
        public int RequestId { get; set; }
```

Reply:
Changed